docs/modules/ROOT/pages/documentation/payara-server/server-configuration/ssl-certificates.adoc

```diff
@@ -66,6 +66,43 @@ In a default installation, the _http-listener-2_ is responsible for handling the
 
 The information can also be entered in the Web Administration console, by navigating to the page Configurations -> server-config -> HTTP Service -> HTTP Listeners > http-listener-2.
 
+[[loading-certificates-from-multiple-keystores]]
+== Loading certificates from multiple keystores
+
+_Since Payara Server 5.2021.5_
+
+IMPORTANT: All additional keystores must have the same password as your domain master password. If you want to load more than one additional keystore use ':' as a delimiter between file paths.
```

Comment:
I'd move the bit about loading multiple keystores out from here - having two separate points in the same caption strikes me as bad practice.